holmes/interactive.py

```
@@ -2429,6 +2429,14 @@ def get_bottom_toolbar():
                 user_input = initial_user_input
                 initial_user_input = None
             else:
+                if config and config.hooks:
+                    for hook in config.hooks.get("on_waiting_for_input", []):
+                        cmd = hook.get("command")
+                        if cmd:
+                            try:
+                                subprocess.Popen(cmd, shell=True)
```

Ruff (0.15.12)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)